wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -6,8 +6,20 @@ WAGN_SEED_PATH = File.join(
   ENV['DECKO_SEED_REPO_PATH'] || [Cardio.gem_root, 'db', 'seed'], 'new'
 )
 
+def run_card_migration core_or_deck
```

Comment:
MUCH nicer!  we might even consider putting this inside Cardio or something similar...